guides/dev/README.md

```diff
@@ -89,6 +89,15 @@ To stop using the virtual environment, run `deactivate`.
 
 ## Install
 
+You likely will need to update/install `pip` prior to installing the development dependencies. To update/install `pip` use the following:
+
+```sh
+# To install pip:
+(.venv) $ python3 -m ensurepip --upgrade
+# To upgrade pip:
+(.venv) $ python3 -m pip install --upgrade pip
+```
+
```

Comment:
trivial: keep it brief, or the reader's eyes glaze over. You said "install pip" like 3 times, separate from the command itself. And you don't need to point out that they need to do this before the next step. People understand that this is a list of sequential steps

I wasn't really aware of ensurepip. That's cool! Let's just always use that

```suggestion
Ensure `pip` (Python's package manager) is installed and up to date:
```sh
(.venv) $ python3 -m ensurepip --upgrade
```

```